packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -144,6 +188,10 @@ export class BedrockInvokeModel extends sfn.TaskStateBase {
       throw new Error('Output S3 object version is not supported.');
     }
 
+    if (props.guardrailConfiguration && props.contentType && (props.contentType !== 'application/json')) {
```

Comment:
> You don't provide a guardrail identifier but you specify the amazon-bedrock-guardrailConfig field in the request body.

Can we check this as well?

Also, don't forget to check if the attributes are resolved before applying validation.
Otherwise, we might block valid deployments.

Reply:
@lpizzinidev 
Thank you. 
Regarding `amazon-bedrock-guardrailConfig`, after reviewing the [documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html), it doesn't seem to be supported in any of the foundation models yet.

I tried configuring it in the management console and running InvokeModel, but it was treated as an invalid value and couldn't be executed.

Since it's not supported yet, I thought it might be better to avoid adding validation for it. What do you think?